src/parameters.toml

```diff
@@ -511,9 +511,9 @@ type = "float"
 
 [epoch_time]
 alias = "epoch"
-value = 211813488000
-type = "float"
-description = "derived: 2451545.0 * [day]"
+value = "2000-01-01T11:58:56.816"
+type = "String"
+description = "J2000 epoch (Jan 1, 2000 11:58:56.816 UTC) as string"
```

Comment:
@odunbar @charleskawczynski Is this compatible with the new format?

Reply:
Right now we have functionality to allow `AbstractFloat` and `AbstractVector{AbstractFloat}`.
Using the type parameter we could add functionality to add reading of `string` type. Would require a PR.

Reply:
If I stored it as a vector `[2000, 01, 01, 11, 58, 56, 816]` - that would work?